std_execution.bs

```diff
@@ -669,6 +669,58 @@ Sender consumers could be made pipeable, but we have chosen to not do so.
 However, since these are terminal nodes in a pipeline and nothing can be piped after them, we believe a pipe syntax may be confusing as well as unnecessary, as consumers cannot be chained.
 We believe sender consumers read better with function call syntax.
 
+## All awaitables are senders ## {#design-awaitables-are-senders}
+
+Since C++20 added coroutines to the standard, we expect that coroutines and awaitables will be how a great many will choose to express their asynchronous code. However, in this paper, we are proposing to add a suite of asynchronous algorithms that accept senders, not awaitables. One might wonder whether and how these algorithms will be accessible to those who choose coroutines instead of senders.
+
+In truth there will be no problem because all awaitable types automatically model the `typed_sender` concept. The adaptation is transparent and happens in the sender customization points, which are aware of awaitables.
+
+For an example, imagine a coroutine type called `task<T>` that knows nothing about senders. It doesn't implement any of the sender customization points. Despite that fact, and despite the fact that the `this_thread::sync_wait` algorithm is constrained with the `typed_sender` concept, the following would compile and do what the user wants:
+
+```c++
+task<int> doSomeAsyncWork();
+
+int main() {
+  // OK, awaitable types satisfy the requirements for typed senders:
+  auto o = this_thread::sync_wait(doSomeAsyncWork());
+}
+```
+
+Since awaitables are senders, writing a sender-based asynchronous algorithm is trivial if you have a coroutine task type: implement the algorithm as a coroutine. If you are not bothered by the possibility of allocations and indirections as a result of using coroutines, then there is no need to ever write a sender, a receiver, or an operation state.
```

Comment:
> there is no need to ever write a sender

This comes with the proviso that there are sufficient concurrency building-blocks available that you can use.
Not all algorithms can be implemented purely with coroutines. Having said that, most algorithms can be written using coroutines once you have something like `async_scope` that can spawn operations that execute concurrently - allowing structured concurrency algorithms to be built out of the ad-hoc concurrency mechanisms.